dependency_support/com_google_skywater_pdk/BUILD

```diff
@@ -22,3 +24,8 @@ licenses(["notice"])
 exports_files([
     "liberty_wrapper.py",
 ])
+
+extract_lef_and_liberty(
+    name = "sky130-fd_sc_hd_data",
```

Comment:
I tried to be consistent with asap7 (`sky130-` instead of `sky130_`) but this is a bit inconsistent with the cell library naming (`sky130_`) let me know which one your prefer.